src/main/java/ru/yandex/practicum/filmorate/controller/GenreController.java

```diff
@@ -0,0 +1,37 @@
+package ru.yandex.practicum.filmorate.controller;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import ru.yandex.practicum.filmorate.model.Genre;
+import ru.yandex.practicum.filmorate.service.GenreService;
+
+import java.util.List;
+
+
+@Slf4j
+@RestController
+@RequestMapping("/genres")
+@AllArgsConstructor
+public class GenreController {
+
+    private final GenreService genreService;
+
+    @GetMapping
+    @ResponseBody
```

Comment:
https://www.baeldung.com/spring-request-response-body
`Remember, we don't need to annotate the @RestController-annotated controllers with the @ResponseBody annotation since Spring does it by default.`

Такую аннотацию можно не ставить на спринговых контроллерах